applications/e-commerce/home.html

```diff
@@ -2,11 +2,26 @@
 <html>
 
 <head>
-
+ <H5>Top 10 colors</H5><hr>
 </head>
 
 <body>
-    Hi class! almost done with git
-    
+    <ol>
+        <li>orange</li>
+        <li>purple</li>
+        <li>black</li>
+        <li>grey</li>
+        <li>blue</li>
+        <li>red</li>
+        <li>yellow</li>
+        <li>white</li>
+        <li>brown</li>
+        <li>Green</li>
+    </ol>
+    <p><h2>do you agree yes or no?<input></h2></p>
+    Why?<textarea name="Your opinion" id="" cols="30" rows="10"></textarea>
```

Comment:
There's too much going on here for me to write about, but we can discuss it in class.